tests/conftest.py

```diff
@@ -29,11 +29,18 @@
 XY_TO_PIR_RESPONSES = sorted(XY_TO_PIR_FOLDER.iterdir())
 
 
+@pytest.fixture()
+def data_folder():
+    """Return data folder."""
+    return DATA_FOLDER
+
+
 @pytest.fixture(autouse=True)
 def custom_cache_dir(monkeypatch, tmpdir):
     # Automatically use a custom cache directory for all tests to avoid writing
     # cache data to the user's real cache.
     monkeypatch.setenv("XDG_CACHE_HOME", str(tmpdir))
+    return pathlib.Path(tmpdir) / "atldld"
```

Comment:
I think this is not necessary.

If you want to actually have this folder inside of tests you can just call `atldld.config.user_cache_dir` and you should get it. This works since it is reading from the `XDG_CACHE_HOME`.

Reply:
Good point. Done in cae39ef

Reply:
Perfect!